client.py

```diff
@@ -0,0 +1,121 @@
+""" run a client that connects to a server and interacts with user using tkinter """
+import constants
+from cryptography.fernet import Fernet
+import re
+import requests
+import tkinter as tk
+
+
+def encrypt_message(message: str) -> bytes:
+    """
+    Encrypts a message
+    :param message: a regular message
+    :return: the encrypted message
+    """
+    key = constants.key
+    f = Fernet(key)
+    message = message.encode()
+    encrypted_message = f.encrypt(message)
+    return encrypted_message
+
+
+def decrypt_message(encrypted_message: bytes) -> bytes:
+    """
+    Decrypts an encrypted message
+    :param encrypted_message: the encrypted message to decrypt
+    :return: the decrypted message
+    """
+    key = constants.key
+    f = Fernet(key)
+    decrypted_message = f.decrypt(encrypted_message)
+    return decrypted_message
+
+
+def download(outputlabel: tk.Label) -> None:
+    """
+    download a requested file from the server's files to a requested location
+    :param outputlabel: the label where that response of server is written
+    """
+    path = constants.request
+    path = path.split('/')
+    filename = path[-1]
+    filename = encrypt_message(filename)
+    with requests.put(f"http://{constants.ip}:{constants.port}/download/", data=filename, stream=True) as response:
+        path = constants.request.split("- ")
+        path = path[-1]
+        with open(str(path), 'w') as f:
+            for chunk in response.iter_content(chunk_size=8192):
+                f.write(decrypt_message(chunk).decode())
+    outputlabel['text'] = f"file downloaded successfully to {path}"
+
+
+def upload(outputlabel: tk.Label) -> None:
+    """
+    upload a requested file to the server
+    :param outputlabel: the label where that response of server is written
+    """
+    path = constants.request.split("- ")
+    path = path[-1]
+    with open(path, 'r') as file:
+        request = constants.request.split("/")
+        name = request[-1]
+        start = True
+        for chunk in file:
+            if start == True:
```

Comment:
```suggestion
            if start is True:
```
Boolean assertion can be with `is` instead of `==`.